assets/dags/mwaa_dr/v_2_4/dr_factory.py

```diff
@@ -182,6 +182,7 @@ def dag_run(self, model: DependencyModel[BaseTable]) -> BaseTable:
                 "start_date",
                 "state",
             ],
+            export_filter="dag_id != 'backup_metadata'",
```

Comment:
Filters backup_metadata to ensure the scheduler does not resume backup_metadata with the status running during batch imports of metadata. If the scheduler resumes, it will overwrite the metadata while the import is running.